### PR TITLE
doc: Reviewed spanish translations

### PR DESCRIPTION
**Pull request by mmadariaga**
Some fixed spanish translations

### CODE REVIEW

File: doc/sphinx/locale/es/LC_MESSAGES/internal_calls.po [hunk 1]
```diff
@@ -445,7 +445,7 @@ msgid ""
 "than the first one)."
 msgstr "Los media-relays se organizan en grupos con el fin de poder asignar un "
 "grupo concreto a una empresa concreta. Cada elemento del grupo tiene una "
-"**métrica** que permite repartos de carga desiguales dentro de un mismo "
+"**métrica** (péso) que permite repartos de carga desiguales dentro de un mismo "
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/internal_calls.po]: You mean *peso* (as weight) here

File: doc/sphinx/locale/es/LC_MESSAGES/external_incoming_calls.po [hunk 2]
```diff
@@ -664,9 +664,9 @@ msgid ""
 "messages to destination **SIP Proxy**, they will be sent to the IP:PORT "
 "of this field."
 msgstr "Normalmente se deja vacío o se pone la IP del dominio indicado en"
-"**SIP Proxy** (para evitar resolución de dominios y hacer que el mensaje"
+"**SIP Proxy** (para evitar resolución de dominios y hacer que el mensaje "
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/external_incoming_calls.po]: An extra space at the beginning of this line or at the end of the previous one will make the bold SIP Proxy work as expected.

File: doc/sphinx/locale/es/LC_MESSAGES/external_outgoing_calls.po [hunk 3]
```diff
@@ -147,7 +147,7 @@ msgstr "Conmutación por error"
 #: ../../external_outgoing_calls/call_routing.rst:73
 msgid "Failover route lets us use another route whenever the main route fails."
 msgstr "Las rutas en caso de fallo o *failover-routes* nos permite disponer de "
-"rutaadicional en caso de que la ruta preferida falle."
+"ruta adicional en caso de que la ruta preferida falle."
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/external_outgoing_calls.po]: I think the original translation for this one will make more sense with *otra ruta adicional* or *una ruta adicional*

File: doc/sphinx/locale/es/LC_MESSAGES/installation.po [hunk 4]
```diff
@@ -378,8 +378,8 @@ msgid ""
 msgstr "Puedes instalar cuantas instancias desees de cada perfil, pero ten "
 "encuenta que, mientras algunos estan pensados para escalar "
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/installation.po]: extra space for *en cuenta*

File: doc/sphinx/locale/es/LC_MESSAGES/installation.po [hunk 5]
```diff
@@ -378,8 +378,8 @@ msgid ""
...
 "horizontalmentede manera nativa (por ejemplo: asterisk o media-relays) "
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/installation.po]: extra space for *horizontalmente de*

**Comment by Kaian** [comment]
I have added some comments. Most of them are not related with the changes of the PR but I've seen them in the context while reading your fixes.

If you can update the PR with those changes it will be awesome.

Thanks Mikel :slightly_smiling_face:

File: doc/sphinx/locale/es/LC_MESSAGES/external_incoming_calls.po (hunk 2 above)
Comment (mmadariaga) [inline doc/sphinx/locale/es/LC_MESSAGES/external_incoming_calls.po]: Thanks. I'll change some other translations affected by the same issue as well

File: doc/sphinx/locale/es/LC_MESSAGES/installation.po (hunk 4 above)
Comment (mmadariaga) [inline doc/sphinx/locale/es/LC_MESSAGES/installation.po]: Although github doesn't show it, this has been fixed in https://github.com/irontec/ivozprovider/pull/40/commits/e80717d59516934cc354d9aa2a36ebe0566ee9af

**Pull request merged**